clients/java/build.gradle.kts

```
@@ -1,22 +1,83 @@
+import org.jreleaser.model.Active
+
+plugins {
+    id("org.jreleaser") version "1.19.0"
+    id("base")
+}
+
+fun readVersion(): String {
```

Saw @Datron's comment - @ShreyBana  - are we leaving reading from Cargo.toml here for fallback - should we just fail if the VERSION env is not defined?